### PR TITLE
vision conversion for tap changer

### PR DESCRIPTION
**Pull request by nitbharambe**
Vision Conversion for automatic tap changer

TODOs:
 - [x] Update validation tests
 - [x] update pgm version

### CODE REVIEW

File: tests/unit/utils/test_uuid_excel_cvtr.py [hunk 1]
```diff
@@ -13,14 +13,17 @@
 DATA_DIR = Path(__file__).parents[2] / "data" / "vision"
 SOURCE_FILE = DATA_DIR / "vision_97_en.xlsx"
 REFERENCE_FILE = DATA_DIR / "vision_en.xlsx"
+MAPPING_FILE_DIR = Path(__file__).parent.parent.parent.parent / "src/power_grid_model_io/config/excel"
```
Comment (Jerry-Jinfeng-Guo) [inline tests/unit/utils/test_uuid_excel_cvtr.py:16]: Maybe:
```suggestion
SRC_CONFIG_DIR = Path("src") / "power_grid_model_io" / "config" / "excel"
MAPPING_FILE_DIR = Path(__file__).parent[3] / SRC_CONFIG_DIR 
```

Comment (Jerry-Jinfeng-Guo) [reply tests/unit/utils/test_uuid_excel_cvtr.py]: Oops, this typo is on me

File: docs/converters/vision_converter.md [hunk 2]
```diff
@@ -44,6 +44,8 @@ Some components are yet to be modeled for conversions because they might not hav
 - The source bus in power-grid-model is mapped with a source impedance. `Sk"nom`, `R/X` and `Z0/Z1` are the attributes used in modeling source impedance. In Vision, these attributes are used only for short circuit calculations
 - The load rate for transformer is calculated in Vision by current i.e., `load_rate = max(u1 * I1, u2 * I2) * sqrt(3) / Snom * 100`. Whereas in power-grid-model, loading is calculated by power, i.e., `loading = max(s1,s2)/sn`. (Note: The attribute names are as per relevant notation in Vision and PGM respectively). This gives a slight difference in load rate of transformer.
 - A minor difference in results is expected since Vision uses a power mismatch in p.u. as convergence criteria whereas power-grid-model uses voltage mismatch.
+- The Voltage Control option in vision is modelled by `transoformer_tap_regulator` in PGM. Currently, PGM only supports line drop compesnation on both loading and generating direction.
+So the unchecking of `Also in backw. direction`. `Load Dependent` compensation and a master-slave configuration for control under the `State` section are both not supported.
```
Comment (Jerry-Jinfeng-Guo) [inline docs/converters/vision_converter.md:48]:
```suggestion
So the unchecking of `Also in backw. direction`. Neither `Load Dependent` compensation or a master-slave configuration for control under the `State` section is supported.
```

**Review by mgovers — COMMENTED** [review]
Only this is left. Fine with everything else

**Review by Jerry-Jinfeng-Guo — APPROVED** [review]

**Pull request merged**